classes.js

```diff
@@ -0,0 +1,8 @@
+ class edwin{
+    constructor(eye,phone){
+        this.eye=eye;
+        the.phone=phone;
```

Comment:
line 4 should be `this.phone = phone`  and class name in upperCase 

Just stalking don't mind me

Reply:
Thanks Olivier, am sure he will make corrections when he sees this.

Reply:
I am just that stalker on Github, You can't imagine how much time I spent here reading people's code. 🙈🙈🙈

Reply:
noted